app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
```

Comment:
I would like to suggest leaving this variable in. Will and I built a buttons component yesterday, it was three buttons side by side with a slight border between them. We wanted that border to match the background, as if it was knocked out. This variable was exactly what we needed.